api-reference/authentication/oauth2-migration-best-practices.markdown

```diff
@@ -73,6 +73,6 @@ layout: reference
     - for a full list, review this doc: https://developer.concur.com/api-reference/authentication/apidoc.html#response_codes
 
 ### 4. Old auth v.s. new auth diagram
-![old v.s. new](https://raw.githubusercontent.com/concur/developer.concur.com/preview/api-reference/authentication/newAndOldAuthComparison.png)
+![old v.s. new](https://raw.githubusercontent.com/zenan-angus-wang/developer.concur.com/preview/api-reference/authentication/oldNewAuthComparion.png)
```

Comment:
I should've caught this in the last PR, but you can reference the image using a relative path, which is preferred.

```
![old vs. new](./oldNewAuthComparion.png)
```